docs/source/index.rst

```diff
@@ -1,81 +1,51 @@
 |PyPI| |Bioconda| |Downloads| |CI| |Notebooks| |Docs| |Codecov|
 
-CellRank for directed single-cell fate mapping
-==============================================
+CellRank: dynamics from multi-view single-cell data
```

Comment:
Don't really like the new title

Reply:
I changed it to not be identical with our first paper; I think that's fine for now.